chatgpt-plugin/src/pages/api/handlers/solana-pay/sign/testCreateMintNFT.sh

```diff
@@ -0,0 +1,8 @@
+curl --location 'localhost:3000/api/handlers/solana-pay/sign/createMintNFT?=null' \
```

Comment:
Please remove this file, and instead provide the curl as a comment for now. 

Let's add tests for this in a separate PR

Reply:
@govarun please provide the curl command in the description of this PR, instead of comment. My bad.